Bottom: 80acc0d6c48ab59292ae491776930a6d6c99ff32
Top:    f6e5d722efdd096b7c1c146c7693eadc97050f0f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-01-25 20:52:03 +0000

Fix linking with newer LD


---

diff --git a/configure.ac b/configure.ac
index d7bb9b4..27be6d2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -430,6 +430,9 @@ Either do not use --enable-gl or enable the gtk HID.])
 	  * ) ;;
 	esac
 
+	GLU_CFLAGS=
+	GLU_LIBS=-lGLU
+
 	AC_DEFINE([ENABLE_GL], 1,
 		[Define to 1 if GL support is to be compiled in])
 fi
@@ -1066,8 +1069,8 @@ AM_CONDITIONAL(DEBUG_BUILD, test x$enable_debug = xyes)
 
 # ------------- Complete set of CFLAGS and LIBS -------------------
 
-CFLAGS="$CFLAGS $X_CFLAGS $DBUS_CFLAGS $GLIB_CFLAGS $GTK_CFLAGS $CAIRO_CFLAGS $GTKGLEXT_CFLAGS"
-LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_LIBS $INTLLIBS $CAIRO_LIBS $GTKGLEXT_LIBS"
+CFLAGS="$CFLAGS $X_CFLAGS $DBUS_CFLAGS $GLIB_CFLAGS $GTK_CFLAGS $CAIRO_CFLAGS $GTKGLEXT_CFLAGS $GLU_CFLAGS"
+LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_LIBS $INTLLIBS $CAIRO_LIBS $GTKGLEXT_LIBS $GLU_LIBS"
 
 
 # if we have gcc then add -Wall
